darwin/exporter/formats/darwin_1_0.py

```diff
@@ -0,0 +1,134 @@
+from darwin.version import __version__
+
+
+def export(annotation_files: Iterable[dt.AnnotationFile], output_dir: Path) -> None:
+    for id, annotation_file in enumerate(annotation_files):
+        _export_file(annotation_file, id, output_dir)
+
+
+def _export_file(annotation_file: dt.AnnotationFile, id: int, output_dir: Path):
+    output: Dict[str, Any] = _build_json(annotation_file)
+    output_file_path: Path = (output_dir / annotation_file.filename).with_suffix(".json")
+    with open(output_file_path, "w") as f:
+        json.dump(output, f, cls=NumpyEncoder, indent=1)
+
+
+def _build_json(annotation_file: dt.AnnotationFile):
+    if annotation_file.is_video:
+        return _build_video_json(annotation_file)
+    else:
+        return _build_image_json(annotation_file)
+
+
+def _build_image_json(annotation_file: dt.AnnotationFile):
+    return {
+        "image": {
+            "seq": annotation_file.seq,
+            "width": annotation_file.image_width,
+            "height": annotation_file.image_height,
+            "filename": annotation_file.filename,
+            "original_filename": annotation_file.filename,
+            "url": annotation_file.image_url,
+            "thumbnail_url": annotation_file.image_thumbnail_url,
+            "path": annotation_file.remote_path,
+            "workview_url": annotation_file.workview_url,
+        },
+        "annotations": list(map(_build_annotation, annotation_file.annotations)),
+    }
+
+
+def _build_video_json(annotation_file: dt.AnnotationFile):
+    return {
+        "image": {
+            "seq": annotation_file.seq,
+            "frame_urls": annotation_file.frame_urls,
+            "frame_count": len(annotation_file.frame_urls or []),
+            "width": annotation_file.image_width,
+            "height": annotation_file.image_height,
+            "filename": annotation_file.filename,
+            "original_filename": annotation_file.filename,
+            "thumbnail_url": annotation_file.image_thumbnail_url,
+            "url": annotation_file.image_url,
+            "path": annotation_file.remote_path,
+            "workview_url": annotation_file.workview_url,
+        },
+        "annotations": list(map(_build_annotation, annotation_file.annotations)),
+    }
+
+
+def _build_annotation(annotation):
+    if isinstance(annotation, dt.VideoAnnotation):
+        return _build_video_annotation(annotation)
+    else:
+        return _build_image_annotation(annotation)
+
+
+def _build_author(author: dt.AnnotationAuthor) -> Dict[str, Any]:
+    return {"full_name": author.name, "email": author.email}
+
+
+def _build_sub_annotation(sub: dt.SubAnnotation) -> Dict[str, Any]:
+    if sub.annotation_type == "instance_id":
+        return {sub.annotation_type: {"value": sub.data}}
+    elif sub.annotation_type == "attributes":
+        return {sub.annotation_type: sub.data}
+    elif sub.annotation_type == "text":
+        return {sub.annotation_type: {"text": sub.data}}
```

Comment:
we are missing `directional vectors`, `measures` and `inference`

Reply:
I see no mention of any of them anywhere in darwin-py. I've added generic clause, because I think importer does that too already. I hope this handles those cases - could you take a look during your functional pass? I'll do the same if I find out how to create those.